File: 5.2.py
def powr(a,b):
    m=1
    for i in range(b ) :
        m=m*a
    return m 
a=int(input("enter the number \n"))
b=int (input ("then power :\n"))
print (f" the result is : {powr(a,b)}")

File: 5.2.c
#include<stdio.h>
int powr(int a , int b){
    int m=1;
    for (int i=1;i<=b;i++){
     m=a*m;
    }return m;
}
int main(){
    int a,b;
    printf("enter a and b that (a^b):\n");
    scanf("%d %d",&a,&b);
    printf ("the resulte : %d",powr(a,b));
    return 0;
}